token/cli/src/config.rs

```diff
@@ -31,12 +31,12 @@ pub(crate) struct MintInfo {
 }
 
 pub(crate) struct Config<'a> {
-    pub(crate) default_signer: Arc<dyn Signer>,
+    pub(crate) default_signer: Option<Arc<dyn Signer>>,
```

Comment:
To be totally sure that this is being used correctly, can you make this field private?
```suggestion
    default_signer: Option<Arc<dyn Signer>>,
```

Reply:
Ah right that makes sense.  In that case, I'd say don't worry about it. I'll retract these